src/http/cookie.cr

```diff
@@ -27,6 +27,7 @@ module HTTP
     property extension : String?
     property max_age : Time::Span?
     getter creation_time : Time
+    property? partitioned : Bool
```

Comment:
This whole type could use a doc pass. Going to just do all that at once in a follow up PR.